functional/start_etcd

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+USER_ID=${SUDO_UID:-$(id -u)}
+HOME=$(getent passwd "${USER_ID}" | cut -d: -f6)
```

Comment:
Temp will be cleaned up after reboot. And after reboot IP address could be also changed. This script also works under root user.